src/main/java/com/google/sps/servlets/ReviewServlet.java

```diff
@@ -180,6 +179,18 @@ public Entity trimQuery(List<Entity> queryResults) /*throws IOException*/ {
     return queryResults.get(0);
   }
 
+  /** Aggregate Scores of all Comments in location
+   *  Currently listed in this servlet, but option to move to class.
+   */
+  public double aggregateScore(List<Comment> reviews) {
```

Comment:
Does it make sense to have this function in this servlet? You currently don't call this function, but I imagine it will be called in doGet() after retrieving all user reviews in a location which, essentially, means we'll be recalculating the aggregate score each time a user clicks on a search result. While the computations here should be small in number, I think we can do better if we instead use our placesServlet to keep track of (int) totalReviews and (double) total_sum_of_scores so that each new review can just be added to these two and we can easily retrieve total_sum_of_scores / totalReviews instead of looping through them each time. Alternatively, an easier-to-implement solution that will still prevent us from loop through a second time is simply calculating these two values as we collect them in the doGet() in the first place so that we handle everything in a single loop.

Reply:
Thanks Erick. I think for now I'll just compute the scores as we go along. Although costly to calculate every time a place is queried, I think it will be easier for right now.